src/RaisedButton/RaisedButton.js

```diff
@@ -134,6 +134,13 @@ class RaisedButton extends Component {
      */
     className: PropTypes.string,
     /**
+     * This is the Link tag that can be used inside RaisedButton.
```

Comment:
Made a change to that comment. I am still not sure about wording the documentation for containerElement.